features/financial_assistance/step_definitions/disable_medicaid_tax_credits_link_steps.rb

```diff
@@ -13,11 +13,21 @@
 end
 
 Then(/^they should see the Medicaid and Tax Credits Link tile$/) do
-  expect(page).to have_content("Medicaid & Tax Credits")
+  expect(page).to have_content(l10n("insured.families.medicaid_and_tax_credits"))
```

Comment:
let's assert on the element that _contains_ this title rather than the content itself